packages/wp-story-editor/src/components/postLock/stories/postTakeOverDialog.js

```diff
@@ -27,21 +22,26 @@ import PostTakeOverDialog from '../postTakeOverDialog';
 export default {
   title: 'Stories Editor/Components/Dialog/Post Take Over Dialog',
   component: PostTakeOverDialog,
-};
-
-export const _default = () => {
-  const user = {
+  args: {
+    isOpen: true,
     name: 'Matt Mullenweg',
     avatar:
       'http://1.gravatar.com/avatar/767fc9c115a1b989744c755db47feb60?size=48',
+    dashboardLink: 'http://www.example.com/dashboard',
+  },
+  argTypes: {
+    onClose: { action: 'onClose' },
+  },
+  parameters: {
+    controls: { exclude: ['user'] },
+  },
+};
+
+export const _default = (args) => {
+  const user = {
+    name: args.name,
+    avatar: args.avatar,
```

Comment:
💭  should `name` and `avatar` be removed from the `args` object so they aren't passed into `PostTakeOverDialog`?

Reply:
Or maybe we could update args to be
```js
args: {
    user: {
        name: 'Matt Mullenweg',
        avatar: 'avatar-url'
    }
}
```

Reply:
Totally. I only broke it out cause I like the string input ui on controls over editing via an object. BUT the object is more true to form. 
Updated.
<img width="945" alt="Screen Shot 2022-02-07 at 1 59 45 PM" src="https://user-images.githubusercontent.com/1820266/152871100-ae9d3c8c-f94b-4713-ad04-cb285aa41f55.png">